### DISCUSSION TITLE
DB Backup Fails

### DISCUSSION THREAD

**Discussion by tebulrich** [spatie/laravel-backup#1557] [Q&A]
Environment is in a docker container
MySQL version 8.0.27

Starting backup...
Dumping database laravel-project...
Backup failed because The dump process failed with a none successful exitcode.
Exitcode
========
2: Misuse of shell builtins

Output
======
<no output>

Error Output
============
mysqldump: Got error: 1045: "Plugin caching_sha2_password could not be loaded: /usr/lib/x86_64-linux-gnu/mariadb19/plugin/caching_sha2_password.so: cannot open shared object file: No such file or directory" when trying to connect
.
#0 /var/www/html/vendor/spatie/db-dumper/src/DbDumper.php(178): Spatie\DbDumper\Exceptions\DumpFailed::processDidNotEndSuccessfully(Object(Symfony\Component\Process\Process))

**Comment by angeljqv** [comment]
you are missing a dependency

**Reply by angeljqv** [reply]
>and which one would that be?

[Plugin caching_sha2_password could not be loaded](https://www.google.com/search?q=mysqldump+Plugin+caching_sha2_password+could+not+be+loaded&oq=mysqldump+Plugin+caching_sha2_password+could+not+be+loaded&aqs=chrome..69i57.1409j0j1&sourceid=chrome&ie=UTF-8)

**Comment by tebulrich** [comment]
and which one would that be? I just used composer require spatie/laravel-backup which should install all requirements (except the zip extension for PHP, which is installed)

**Comment by lucianobosco** [comment]
I've just installed this package and as soon I try to run the backup I face the same error.
I'm using mysql:8.0.30 so I'm wondering if this package is not compatible with latest MySQL releases.
Any advice will be appreciated
